playbooks/keyholders/EXISTING_SIGNER.md

```diff
@@ -0,0 +1,109 @@
+# New Keyholder Responsibilities
+
+This document outlines the responsibilities of an existing keyholder.
+
+## Outline
+
+Thank you for being a Sigstore Root Keyholder!
+
+Make sure you have read over the keyholder responsibilities in [OVERVIEW.md](./OVERVIEW.md). This document will cover [testing](#testing) and the actual ceremony [signing](#ceremony-signing) to perform during the ceremony.
+
+If you are testing with a new key, follow [New Signer Testing](./NEW_SIGNER.md/#testing).
+
+## Testing
+
+Complete the following steps in order:
+
+### Pre-requisites
+
+Ensure you have the following:
+- [ ] A local Git installation and a Go development setup
+- [ ] SSH authentication for GitHub (see [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
+- [ ] A USB port connection for your hardware key (beware of using a remote connection; the keyholder should not assume that magic occurs during an SSH session)
+- [ ] A fresh environment. In particular, ensure that environment variables like `LOCAL`, `GITHUB_USER`, and `BRANCH` are unset before you begin.
+- [ ] A fork of the [root-signing](https://github.com/sigstore/root-signing) repository. Click the "fork" button on GitHub and clone the repository.
+
+### Setup 
+
+- [ ] Test binary build: Set your `${GITHUB_USER}` with your GitHub username and execute the script:
+```bash
+export GITHUB_USER=${GITHUB_USER}
+export LOCAL=1
+./scripts/step-0.sh
+```
+This will setup your fork and build the TUF binary to use for metadata generation. This will also disable PR creations after each step and allow you to test changes locally.
```

Comment:
```suggestion
This will setup your clone and build the TUF binary to use for metadata generation. This will also disable PR creations after each step and allow you to test changes locally.
```